### PR TITLE
Removed unwanted logs .

### PR DESCRIPTION
**Pull request by deepak365**

### CODE REVIEW

**Comment by sam6230i** [comment]
LGTM

File: build.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <property name="lib.dir" value="lib" />
   <property name="build.dir" value="build" />
   <property name="project.name" value="insta4j" />
-  <property name="project.version" value="0.2.16" />
+  <property name="project.version" value="0.2.17" />
```
Comment (asdivekar) [inline build.xml]: @deepak365 oOne question. Do we update the version for every change in the jar?

**Review by aniketdivekar — APPROVED** [review]

File: build.xml (hunk 1 above)
Comment (deepak365) [inline build.xml]: @asdivekar : yes.

**Comment by kidsid49** [comment]
Guys,

Remove my mail Id from this :)

- Siddarth

**Pull request merged**